scripts/autoparser.sh

```diff
@@ -0,0 +1,2 @@
+make autoparser
+build/autoparser
```

Comment:
We don't need to compile `autoparser` separately. Having a `run` and `test` script should be enough